src/main/java/com/icthh/xm/tmf/ms/activation/domain/SagaEvent.java

```diff
@@ -17,10 +17,12 @@
 import javax.persistence.Id;
 import javax.persistence.Table;
 import lombok.Data;
+import lombok.ToString;
 import lombok.experimental.Accessors;
 
 @Data
 @Accessors(chain = true)
+@ToString(exclude = "taskContext")
```

Comment:
pls add log that write taskContext on creation, for ability to find event id by context

Reply:
context data can be user-sensitive, I have a doubt the we really need print it in logs
@amedvedchuk  what do you think?

Reply:
maybe lets print ID only?

Reply:
Id of event and transaction still wil be printed to logs

Reply:
context in log it's important to find issues in existing system
maybe we need add some configuration to configure this behavior?

Reply:
Than let's plan additional session and discuss this MR

Reply:
+